### PR TITLE
Refactor MLlib build and folder structure.

### PR DESCRIPTION
**Pull request by mengxr**
* Version names were renamed from `onepoint*` to `v1p*`.
* Build script was updated to use `-Dspark.version=` to specify a Spark version.
* `onepoint*` is removed from package names. We don't have any dependency between those sub-projects. They are share the same package/class name. It is easier to create a new sub-project with copy & paste.

@brkyvz

### CODE REVIEW

File: mllib-tests/v1p2/src/main/scala/mllib/perf/LinearAlgebraTests.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package mllib.perf.onepointone
+package mllib.perf
```
Comment (brkyvz) [inline mllib-tests/v1p2/src/main/scala/mllib/perf/LinearAlgebraTests.scala]: I think this is in the wrong folder. The path says v1p2, these used to be 1.1

File: mllib-tests/v1p2/src/main/scala/mllib/perf/StatTests.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package mllib.perf.onepointone
+package mllib.perf
```
Comment (brkyvz) [inline mllib-tests/v1p2/src/main/scala/mllib/perf/StatTests.scala]: Ditto, path says v1p2, while it used to be 1.1

**Comment by brkyvz** [comment]
LGTM. Fingers crossed for the tests

**Comment by mengxr** [comment]
Merged.

**Pull request closed without merging**